packages/inventory/src/DeleteModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    Modal,
+    Split,
+    SplitItem,
+    Stack,
+    StackItem,
+    Level,
+    LevelItem,
+    Button,
+    ClipboardCopy
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from  '@patternfly/react-icons';
+
+const DeleteModal = ({ handleModalToggle, isModalOpen, currentSytem, onConfirm }) => {
+    let systemToRemove;
+    let systemLabel = 'system';
+    if (Array.isArray(currentSytem)) {
```

Comment:
According to @karelhala comment below, it should be enough to also leave only the else path

Reply:
Not really, here we want to allow multiple systems for deletion, so we can reuse this component on inventory table.

Reply:
Discussed it with @karelhala and if I understand, DeleteModal component should be also used in the inventory table, where the isArray option will be needed